geotrek/common/mixins/models.py

```diff
@@ -317,6 +319,14 @@ class PublishableMixin(BasePublishableMixin):
     class Meta:
         abstract = True
 
+    def save(self, *args, **kwargs):
+        old_object = self.__class__.objects.filter(pk=self.pk)
+        if (old_object and getattr(old_object, 'review', None) != self.review and self.review) and settings.ALERT_REVIEW:
```

Comment:
you can't acces review attribute from a queryset, you should get instance from qs